editor/src/components/canvas/console-proxy.ts

```diff
@@ -17,6 +17,12 @@ const ConsoleMethodsToProxy: Array<string> = [
   'assert',
 ]
 
+const SuppressedReactMessages = [
```

Comment:
we were printing so much of these React warnings, it made it impossible for me to debug jest tests locally. I wrote a simple filtering for them in console-proxy.